k4FWCore/components/Writer.cpp

```diff
@@ -257,8 +259,15 @@ class Writer final : public Gaudi::Functional::Consumer<void(const EventContext&
         // Check the case when the data has been produced using the old DataHandle
         const auto old_collection = dynamic_cast<DataWrapperBase*>(storeCollection);
         if (!old_collection) {
-          error() << "Failed to cast collection " << coll << endmsg;
-          return;
+          // This can happen for objects that are not collections like in the
+          // MarlinWrapper for converter maps or a LCEvent, or, in general,
+          // anything else
+          warning() << "Object in the store with name " << coll
```

Comment:
Maybe warning is too strong for this. info level or equivalent?

Reply:
I agree, I think warning is too strong here. There is no easy way to filter for certain types here, right? Because in principle for some it is quite expected (e.g. the `LCEvent` from the marlinwrappers) while for others it might be unexpected.

Reply:
Changed to `info`. Checking for types from the Marlin wrapper would make fwcore depend on Marlin wrapper who already depends on fwcore. Anyway this is a message that appears once per type since after the first time it won't be checked again.